### ISSUE TITLE
Understrap Import appears to do nothing 

### ISSUE THREAD

**Issue by hsankala** [understrap/understrap-child#359]
In [src/sass/assets/understrap.scss ](https://github.com/understrap/understrap-child/blob/fa849f7a6a9c4ef885b94873fd66b99e2e872ab4/src/sass/assets/understrap.scss) it imports

[@import "understrap/understrap/understrap";](https://github.com/understrap/understrap-child/blob/fa849f7a6a9c4ef885b94873fd66b99e2e872ab4/src/sass/assets/understrap.scss#L1)

The directory it's pointing to doesn't exist and so it doesn't import anything.

**Comment by bacoords** [comment]
Yeah and I'm not actually sure that the file itself is even being called. 

I'll keep this issue open so that we can clear it up when we pull in the upcoming 1.2 release changes from the parent theme.
